styles.css

```diff
@@ -0,0 +1,166 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600;700&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Poppins", sans-serif;
+  background-color: #bbb7b7;
+}
```

Comment:
- [ ] [REQUIRED] Good job so far! The `background-color` value given to `Body` is reflected on the `right` side in the mobile version, please fix it 😃 

![image](https://user-images.githubusercontent.com/63915024/199279338-6da387ed-cc16-41c8-8151-9eef7426fa6b.png)